### ISSUE TITLE
ac display option to choose background/wallpaper

### ISSUE THREAD

**Issue by sydvaldez** [AChep/AcDisplay#136]
Good day sir iam very very amazed of ac display sir.. i got some suggestion about the the background , i just wonder if possible that we can choose wallpapers or live wallpapers regardless of the homescreen wallpaper that would be more great :)

**Comment by Oozemeister** [comment]
+1

**Comment by CarlosACepeda** [comment]
This won't affect performance? @AChep ?

**Comment by AChep** [comment]
Nah, not much. But… this is fairly hard to implement (especially live wallpapers).

**Comment by CarlosACepeda** [comment]
I guess Live wallpapers are not in the scope of this issue however, hah.
For now it can be just a static wallpaper the user picks from the Gallery.

**Comment by CarlosACepeda** [comment]
My initial question was that each time this image has to be retrieved from storage, in older phones this can be harmful, I guess, because the slowness
